tests/integration_tests/test_integration.py

```diff
@@ -0,0 +1,22 @@
+import json
+
+import pytest
+from pg_to_evalscript import convert_from_process_graph
+
+from tests.utils import get_process_graph_json, run_evalscript
+
+
+@pytest.mark.parametrize(
+    "pg_name,example_input,expected_output",
+    [("test_graph_1", [{"B01": 3, "B02": 3}, {"B01": 5, "B02": 1}], [4, 2])],
```

Comment:
So the idea is to write a bunch of similar tests for each process?

Reply:
Well, this PR is for "integration" tests, so we test the actual evalscript produced based on the process graph, but we'll be able to test other attributes as well (e.g. if the `sampleType` is set correctly, if the number of bands get calculated correctly, if the encoding/decoding works correctly etc.). This may or may not use all the defined processes. 

After this is merged I will create issues for writing all these sorts of tests.